src/main/connector/system/WindowsProcess/WindowsProcess.yaml

```
@@ -34,35 +32,48 @@ monitors:
             ExecutablePath,
             ThreadCount,
             PageFaults,
-            PageFileUsage,
             ReadOperationCount,
             ReadTransferCount,
             WriteOperationCount,
             WriteTransferCount,
-            CommandLine
+            CommandLine,
+            WorkingSetSize,
+            VirtualSize,
+            KernelModeTime,
+            UserModeTime,
+            HandleCount
             FROM Win32_Process
           computes:
           - type: keepOnlyMatchingLines
             column: 2
             regExp: ${var::matchName}
           - type: keepOnlyMatchingLines
-            column: 12
-            regExp: ${var::matchCommand}   
+            column: 11
+            regExp: ${var::matchCommand}
+          - type: add
+            column: 14
+            value: $15
+          - type: divide
+            column: 14 #  unit = 100-nanosecond intervals
+            value: 10000000 # divide by 10000000 to get a result in seconds
       mapping:
         source: ${source::processQuery}
         attributes:
-          id: $12
+          id: $11 # The commandLine is the monitor id

```

It's better and safer to use the PID for the `id` attribute:

```suggestion
          id: $1
```